### PR TITLE
Add new Stiky resin bying Quest

### PR DESCRIPTION
**Pull request by Dream-Master**

### CODE REVIEW

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -307649,6 +307649,72 @@
...
+          "repeatTime:3": 18000,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Sticky matter",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Cant find any Rubber Tree? Can\u0027t build a water tank? Lazy to build a tree tap?  Have a stack of sticky resin for some coins."
```
Comment (Prometheus0000) [inline config/betterquesting/DefaultQuests.json:307682]: Cant find any Rubber Tree? Can\u0027t build a water tank? Lazy to build a tree tap? Have a stack of sticky resin for some coins.
should be
Can\u0027t find any Rubber Trees? Can\u0027t build a water tank? Too lazy to make a tree tap? Buy a stack of sticky resin for some coins.

This assumes the \u0027 means '

File: config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -307649,6 +307649,72 @@
...
+            "Count:3": 1,
+            "Damage:2": 0,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": 18000,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Sticky matter",
```
Comment (Prometheus0000) [inline config/betterquesting/DefaultQuests.json:307677]: The M should be capitalized.

**Review by Prometheus0000 — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@Prometheus0000 can you re check ?

File: config/betterquesting/DefaultQuests.json (hunk 1 above)
Comment (Prometheus0000) [reply config/betterquesting/DefaultQuests.json:307682]: 'tank?Too' should have a space

**Comment by Dream-Master** [comment]
@Prometheus0000 done

**Review by Prometheus0000 — APPROVED** [review]
I can see two spaces before the 'Have', but when I try to copy it it only shows 1, so whatever.

**Comment by Dream-Master** [comment]
image

**Pull request merged**